README.md

```diff
@@ -8,11 +8,13 @@ YJDH-Kesäseteli
 
 1. Copy the contents of `.env.kesaseteli.example` to `.env.kesaseteli` and modify it if needed.
 
-2. Run `yarn kesaseteli:up`
+2. Run `yarn kesaseteli up` or, if you want to rebuild, then `yarn kesaseteli up --build`
 
 The Frontend is now running at [localhost:3000](https://localhost:3000)
 The backend is now running at [localhost:8000](https://localhost:8000)
 
+3. If services fail to get up, `yarn clean` might help.
```

Comment:
should we include clean in the docker?

Reply:
I can fix it to use this https://docs.tibco.com/pub/mash-local/4.1.1/doc/html/docker/GUID-BD850566-5B79-4915-987E-430FC38DAAE4.html